statsmodels/stats/tests/test_lilliefors.py

```diff
@@ -1,4 +1,19 @@
-from numpy.testing import assert_, assert_raises
+'''
+Test Lilliefors corrected K-S tests for normality and exponential distributions
+
+Reference values from R packages:
+    `nortest` (lillie.test)
+    `KScorrect` (LcKS)
```

Comment:
It's better to put this in a comment directly to the test
(in unit tests we use only `#` comments, not triple quotes)
Also adding the full R command or call is helpful for future reference if there are any options. 
(copy-paste into R is easier than figuring out what you might have used.)